### PR TITLE
Install conan and ftxui

### PR DESCRIPTION
**Pull request by gruszczrob**
In this pull request I have installed conan and ftxui. I have set up my workspace to be able to work on next step

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from conan import ConanFile
+from conan.tools.cmake import cmake_layout
+
+
+class CompressorRecipe(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "CMakeToolchain", "CMakeDeps"
+
+    def requirements(self):
+        self.requires("zlib/1.2.11")
```
Comment (ProrokWielki) [inline conanfile.py:10]: you dont need zlib :)

File: CMakeUserPresets.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (ProrokWielki) [inline CMakeUserPresets.json:1]: should not be added to the repo

**Comment by ProrokWielki** [comment]
dont add build dir to the directory :D best to add it to .gitignore

**Comment by gruszczrob** [comment]
Changed everything you said

**Review by ProrokWielki — APPROVED** [review]

**Pull request merged**